### PR TITLE
share first draft

### PR DESCRIPTION
**Pull request by ghost**
clarifying questions are in comments on the md

### CODE REVIEW

File: docs/sources/visualizations/histogram-draft.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
++++
+title = "Histogram"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
+aliases =["/docs/grafana/latest/features/panels/histogram/", "/docs/grafana/latest/panels/visualizations/histogram/"]
+weight = 605
++++
+
```
Comment (achatterjee-grafana) [inline docs/sources/visualizations/histogram-draft.md:8]:
```suggestion
# Visualize Prometheus histogram in Grafana
```

File: docs/sources/visualizations/histogram-draft.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
++++
+title = "Histogram"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
+aliases =["/docs/grafana/latest/features/panels/histogram/", "/docs/grafana/latest/panels/visualizations/histogram/"]
+weight = 605
++++
+# Visualize Prometheus histogram in Grafana
+Assuming users have a basic understanding of [Prometheus](https://grafana.com/oss/prometheus/) and Grafana, this documentation will look at histograms from the perspective of [Grafana 7.0](https://grafana.com/blog/2020/05/18/grafana-v7.0-released-new-plugin-architecture-visualizations-transformations-native-trace-support-and-more/).
```
Comment (Eve832) [inline docs/sources/visualizations/histogram-draft.md:9]: I suggest you use rel refs...if I have time I can try to find the paths, but you can look in the Cloud topics in the `grafana/website` repo for example formatting.

Comment (achatterjee-grafana) [reply docs/sources/visualizations/histogram-draft.md:9]: Yes, this is really a WIP, not ready for review yet.

**Comment by Eve832** [comment]
I think the linking problem is created by the `!` before the URL. This character means that you want to include an image, but you're not providing a path to the image. I'd get rid of them and see if that fixes the build problem.

File: docs/sources/visualizations/histogram-draft.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
++++
+title = "Histogram"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
+aliases =["/docs/grafana/latest/features/panels/histogram/", "/docs/grafana/latest/panels/visualizations/histogram/"]
```
Comment (achatterjee-grafana) [inline docs/sources/visualizations/histogram-draft.md:5]: Remove alias metadata

**Pull request closed without merging**